src/app/presenter/ChessBoardPresenter.ts

```diff
@@ -11,11 +12,28 @@ export class ChessBoardPresenter {
   onSquareWasClicked(position: Position): void {
     //chessModel zwroci mozliwe ruchy dla bierki na wybranej pozycji
     //presenter kaze, zeby view wyswietlil mozliwe ruchy -> wywolujac np. view.showAvailableMoves(squares).
-    console.log(position);
+    this.view.hideSelection();
     this.view.showSelectedPiece(position);
+
+    const squaresStringArray = this.getPossibleMoves(position);
+    console.log('Square array of possible moves: ', squaresStringArray);
```

Comment:
Usuń niepotrzebne console logi :)

Reply:
ja już to zrobiłem, tylko chyba nie pushnąłem jeszcze